topics/plugins-registry.dita

```diff
@@ -180,6 +180,20 @@
 ]</codeblock>
     </section>
     <section id="adding-custom-registries">
+      <p>When you have multiple versions of a plug-in, include a descriptor for each version,
+        separated by a
+        comma:<codeblock>[
+ { "name": "org.example.myplugin",
+   ...
+   "vers": "2.1.0",
+   ...
+ }<b>,</b>
+ { "name": "org.example.myplugin",
+   ...
+   "vers": "1.0.1",
+   ...
+ }
+]</codeblock></p>
```

Comment:
@drmacro This doesn't look valid (or intentional) — perhaps a cursor placement issue in the authoring environment?

Title must be the first element in a section, but the new content is inserted here between the opening `<section>` element and the title.

I assume the intention was to place it at the end of the preceding section?

Reply:
Yes, I didn't intend to split the section. Must have been an issue with the web author not making the new context clear. Or I was just in a hurry.

Reply:
I meant to ask George to add a "Sign commit" to the automatic pull request creation that web author does but I didn't get a chance at the conference.